swagger.json

```diff
@@ -4089,13 +4089,34 @@
                 }],
                 "responses": {
                     "204": {
-                        "description": "Successul response"
+                        "description": "Successful response"
                     },
                     "default": {
                         "$ref": "#/responses/genericError"
                     }
                 }
             }
+        },
+        "/demographics": {
+            "x-swagger-router-controller": "demographics.controller",
+            "get": {
+                "operationId": "listDemographics",
+                "summary": "List demographics",
+                "security": [{
+                    "admin": []
```

Comment:
Do you need clinician here?

Reply:
Nah, this is supposed to only be for Admin viewing for now.